source/fundamentals/connection/tls.txt

```diff
@@ -77,6 +77,26 @@ in the following ways:
    see the :manual:`SRV Connection Format </reference/connection-string/#srv-connection-format>`
    section in the Server manual.
 
+.. note:: Workaround for an "unsafe legacy renegotiation disabled" error
+
+   The {+driver-short+} depends on OpenSSL by default. You may encounter an "unsafe
+   legacy renegotiation disabled" error in certain environments when using OpenSSL 
+   versions 3.0 and later, due to outdated SSL proxies. If you encounter this error, 
```

Comment:
I: [avoid 'due to'](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#d:~:text=Use%20because%20of%20instead.)

```suggestion
   versions 3.0 and later because of outdated SSL proxies. If you encounter this error, 
```

alternatively, you could rephrase the sentence to something like this:

Outdated SSL proxies can cause an "unsafe legacy renegotiation disabled" error in environments
using OpenSSL 3.0 or later.

Reply:
Thanks!